services/viva-ms/lambdas/syncWorkflowAndSetStatus.js

```diff
@@ -31,54 +53,35 @@ async function getAllUserCases(PK) {
     },
   };
 
-  const [error, casesGetResponse] = await to(dynamoDb.call('query', params));
-  if (error) {
-    return console.error('(Viva-ms) syncWorkflow', error);
-  }
-
-  return casesGetResponse;
-}
-
-async function syncCaseWorkflows(cases, personalNumber) {
-  const caseItems = cases.Items;
-
-  for (const caseItem of caseItems) {
-    const workflowId = caseItem.details.workflowId;
-
-    if (!workflowId) {
-      continue;
-    }
-
-    const [getWorkflowError, workflow] = await to(
-      vivaAdapter.workflow.get({
-        personalNumber,
-        workflowId,
-      })
-    );
-    if (getWorkflowError) {
-      return console.error('(Viva-ms) syncWorkflow', getWorkflowError);
-    }
-
-    if (!deepEqual(workflow.attributes, caseItem.details.workflow)) {
-      await syncWorkflowAndStatus(caseItem.PK, caseItem.SK, workflow.attributes);
-    }
-  }
+  return dynamoDb.call('query', params);
 }
 
 async function syncWorkflowAndStatus(PK, SK, workflow) {
```

Comment:
For me this function does many things, if we where to break out the different operations that is done here it would be more readable.

getStatusFromTypeCode
updateCaseWorkflow(PK, SK, workflow, status)